AD container size packages

milljm opened the pull request:
Add MOOSE_OPTIONS to configure

Allow for MOOSE_OPTIONS to be passed as arguments which will be
used during ./configure of MOOSE.

Closes #26300

Commit:
Add MOOSE_OPTIONS to configure

Allow for MOOSE_OPTIONS to be passed as arguments which will be
used during ./configure of MOOSE.

Closes #26300

Commit:
Bump versioner hashes

As always, do this last...

Closes #26300

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/1938964/) on ceff590 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/26303/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/1938979/) on ceff590 wanted to post the following:


## Framework coverage

Coverage did not change


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/geochemistry)
- [`heat_transfer`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/heat_transfer)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/navier_stokes)
- [`optimization`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/optimization)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/scalar_transport)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/26303/coverage/xfem)

</details>



This comment will be updated on new commits.

GiudGiud commented:
MOOSE_CONFIG_OPTIONS? MOOSE_OPTIONS might be too general?

loganharbour commented on scripts/tests/versioner_hashes.yaml:
```diff
@@ -171,3 +171,9 @@ a2e8814e28b97171ff4a06e5343063fed6763dd0: #26102
+  mpich: 702900f
+  petsc: 684d2af
+  libmesh: bad5b94
+  moose-dev: 5b104ea
```
why did moose-dev change?

milljm replied:
Because I modified `moose/conda/moose/*`

milljm replied:
Hmm, does it change every time with a new commit? I just automatically assumed to run the versioner tool. But I see now thats not necessary... I'll change it back.

milljm commented:
> MOOSE_CONFIG_OPTIONS? MOOSE_OPTIONS might be too general?

Because that is the option currently being used elsewhere in other Civet recipes, and this _is_ for Packages/Container-like recipes (Conda MOOSE package creation).
```pre
recipes/moosebuild/moose/branch_next/Apptainer_controlled_apps.cfg:MOOSE_OPTIONS = --with-derivative-size=300
```

Commit:
Remove versioner hash changes

It is not necessary to bump hashes due to a MOOSE package recipe
change, only when a change that will affect the MOOSE package.
